### DISCUSSION TITLE
Incorrect lint:zwave error description

### DISCUSSION THREAD

**Discussion by MartinHjelmare** [zwave-js/zwave-js#7169] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
When running `yarn run lint:zwave` or alternatively using the same tool inside vscode on the fly, the error description when formatting a command class incorrectly doesn't explain what is wrong correctly. It says "Parameter descriptions must be Sentence case".

What did you expect to happen instead?
It should say that command classes must be Title Case.

  1. Add a device config file with options:
  ```json
...
"options": [
  {
    "label": "Central scene mode",
    "value": 2
  }
]
```
  2. Run: `yarn run lint:zwave`
  3. See error:
```
error  Parameter descriptions must be in Sentence case  @zwave-js/consistent-config-string-case
```

### Software versions

Driver (node-zwave-js): ...

Z-Wave JS UI: ...

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: ...

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Checklist

- [ ] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
Converted to issue https://github.com/zwave-js/node-zwave-js/issues/7170
